### DISCUSSION TITLE
badaso-api/v1/maintenance Internal server error 500!

### DISCUSSION THREAD

**Discussion by fkillrra** [uasoft-indonesia/badaso#772] [Q&A] [answered]
hi.

Using docker and curl, I built a demo environment as described in the document.

However, after completing the build, when a login request with ID: "admin@admin.com", PW: "badaso" is requested, an error pop-up saying "Danger key cannot be empty" is displayed.

Any solution?

**Comment by rizkiheryandi** [comment, marked as answer]
We already fix it on latest badaso starter

For existing application, you can run "php artisan jwt:secret"
